packaging/installer/kickstart-static64.sh

```diff
@@ -194,6 +194,7 @@ while [ -n "${1}" ]; do
 		shift 1
 	elif [ "${1}" = "--stable-channel" ]; then
 		RELEASE_CHANNEL="stable"
+		inner_opts="${inner_opts} ${1}"
```

Comment:
Hi @oxplot, are you sure that the inner script accepts the "--stable-channel" option?

Reply:
oh I see — I wrongly assumed based on https://github.com/netdata/netdata/pull/7051#pullrequestreview-300044516 that both scripts behave the same.

I suggest then to only fix kickstart and leave kickstart-static64 for another PR, given how trivial this fix is (so it doesn't drag on).

Reply:
If i may say, @knatsakis if we have decided that through netdata-updater.sh for the static install, we want to automatically set the release channel variable in .environment to stable, when the user uses the stable flag so that updater is automatically pulling stable releases, then it makes sense to add the option. But to make it complete we will need to add handling on the other side too (makeself installer).

If we agree that we do not want to automatically force the stable release channel on the static install, as it was the initial decision with @cakrit when we fixed the updater to work for the static install, then indeed no need to add it to inner opts